docs/HowToUseLWIP.md

```diff
@@ -0,0 +1,111 @@
+# How to use LwIP
+**LwIP (LightWeight IP)** is an open source TCP/IP stack designed for embedded systems.
+Please refer to [lwip_summary](https://savannah.nongnu.org/projects/lwip/) for more details.
+
+## Contents
+> [Main Features: How to Eanble](#main-feature)  
+> [Writing device driver](#device-driver)
```

Comment:
Links are not working.
```
#main-features
#writing-device-driver
```

Reply:
Modification completed. Thanks for comment.